spec/system/visit_index_page_spec.rb

```diff
@@ -106,10 +109,42 @@
     expect(page).to have_content 'MISSING'
   end
 
-  it 'shows the number of expiriing certificates at the top of the page' do
+  it 'shows the number of expiring certificates at the top of the page' do
     first_expiring_certificate = create(:msa_signing_certificate, value: PKI.new.generate_encoded_cert(expires_in: 29.days))
     second_expiring_certificate = create(:msa_signing_certificate, value: PKI.new.generate_encoded_cert(expires_in: 29.days))
     visit root_path
     expect(page).to have_content '2 certificates are expiring soon.'
   end
+
+  it 'shows the expiring certificate lowest expiring certificate 1st' do
+    [
+      create(:msa_signing_certificate, value: PKI.new.generate_encoded_cert(expires_in: 29.days), component: msa_signing_certificate.component),
+      create(:msa_signing_certificate, value: PKI.new.generate_encoded_cert(expires_in: 9.days), component: msa_signing_certificate.component)
+    ].shuffle
+
+    visit root_path
+
+    expires_in_9_days = page.body.index('EXPIRES IN 9 DAYS')
+    expires_in_29_days = page.body.index('EXPIRES IN 29 DAYS')
+    in_use = page.body.index('IN USE')
+
+    expect(expires_in_9_days).to be < expires_in_29_days
```

Comment:
There may be a way to look at the rows in the table on the view and check that the certificate next to expire is at the top, as this expectation seems more unit test like than system spec like. I will have look at how to do it, but not something we need for tomorrow.

Reply:
This was a short term implementation; that worked, there is a more elegant way to do this which would have taken too much time. Which we now have. Thanks for noticing.